tox.ini

```diff
@@ -39,6 +39,8 @@ passenv =
     GALAXY_CONFIG_OVERRIDE_DATABASE_CONNECTION
     APP_WEBSERVER
     TERM
+    TEST_PYTHON
+    TZ
```

Comment:
Are these needed?

Reply:
For local testing, yes. I needed `TZ` for a timezone synonym issue, and `TEST_PYTHON` to easily influence the version of Python tested

Reply:
Doesn't `tox -e py38-test_galaxy_packages` work instead of using `TEST_PYTHON` ?

Reply:
Perhaps if `python3.8` (or `python3.7` in my case) is on your PATH, sure. By adding this, `TEST_PYTHON=~/.pyenv/versions/3.7.12/bin/python tox -e test_galaxy_packages_for_pulsar` will work.